### PR TITLE
Protection against None reference

### PR DESCRIPTION
**Pull request by robeden**
Fixes: #941

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/terraform/runner.py [hunk 1]
```diff
@@ -101,6 +103,8 @@ def check_tf_definition(self, report, root_folder, runner_filter, collect_skip_c
 
     def run_all_blocks(self, definition, definitions_context, full_file_path, root_folder, report,
                        scanned_file, runner_filter, module_referrer: Optional[str]):
+        if not definition:
+            return
```
Comment (schosterbarak) [inline checkov/terraform/runner.py:107]: Thanks for the quick fix. 

I think we would want a debug log in that case.

Comment (robeden) [reply checkov/terraform/runner.py:108]: Broke something. I'm away from my computer right now. I'll look at it when I get back.

**Comment by robeden** [comment]
@schosterbarak @nimrodkor - Something seems to be up with integration tests. Both this and #943 are hitting the same issue and I don’t think either should have any effect on them? I don’t know much about those tests. Any advice?

File: checkov/terraform/runner.py (hunk 1 above)
Comment (robeden) [reply checkov/terraform/runner.py:108]: Looks like something broke in the recent `bc-python-hcl2` release. Error is:
> AttributeError("'Tree' object has no attribute 'items'")

**Comment by schosterbarak** [comment]
thanks for getting this one. just pushed a fix to #944

**Review by tronxd — APPROVED** [review]

**Pull request merged**